get-started/index.md

```diff
@@ -67,42 +67,41 @@ packageserver-8cccc99dd-g7lkh       1/1     Running   0          3m3s
 Use the search bar to find the **Red Hat build of Cryostat** catalog item.
 {% include howto_step.html
   summary="Select the Cryostat Operator and click the Install button"
-  image-name="cryostat-operatorhub-install.png"
+  image-name="`cryostat-operatorhub-install.png`"
 %}
 Choose your Operator installation mode:
-1. In **All Namespaces** installation mode, the Cryostat Operator instance will watch for `Cryostat` or
-`ClusterCryostat` Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
-2. In the **A specific namespace** installation mode, you must also select an installation Namespace, and the Cryostat
-Operator instance will only watch for `Cryostat` or `ClusterCryostat` instances created in that same Namespace.
+1. In **All Namespaces** installation mode, the **Cryostat Operator** instance will watch for **Cryostat** or
+**ClusterCryostat** Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
+2. In the **A specific namespace** installation mode, you must also select an installation Namespace, and the **Cryostat Operator** instance will only watch for **Cryostat** or **ClusterCryostat** instances created in that same Namespace.
 {% include howto_step.html
   summary="Install the Operator"
   image-name="cryostat-operatorhub-install-in-progress.png"
 %}
-Click "Install" and wait for the installation to complete. In this example we will proceed with **All Namespaces**.
+Click "*Install*" and wait for the installation to complete. In this example we will proceed with **All Namespaces**.
 
 Continue to [Setup](#setup).
 
 ## [Setup](#setup)
 
-**Note:** An alternative setup using the multi-namespace `ClusterCryostat` `CR` is described in
-[Alternate Setup](#alternate-setup). For simplicity we will continue with the single-namespace `Cryostat` `CR`.
+**Note:** An alternative setup using the multi-namespace **ClusterCryostat** `CR` is described in
+[Alternate Setup](#alternate-setup). For simplicity we will continue with the single-namespace **Cryostat** `CR`.
 
 ### [Deploying Cryostat](#deploying-cryostat)
 
 {% include howto_step.html
   summary="Create a Cryostat instance"
   image-name="cryostat-operatorhub-install-complete.png"
 %}
-Once the installation is complete, click **Create Cryostat** to create a `Cryostat` `CR` instance in an OpenShift
-Project (Kubernetes `Namespace`) of your choice. This provides configuration information for the Operator to know the
-specifics of how to deploy your Cryostat instance. For full details on how to configure the Cryostat deployment, see
+Once the installation is complete, click ***Create Cryostat*** to create a **Cryostat** `CR` instance in an **OpenShift
```

Comment:
***Create Cryostat*** being both bold and italicized seems inconsistent here - why does this term get both styles applied? According to the precedent set by "*Install*" a few paragraphs higher, it seems like this should follow and be quoted and italicized, since both are talking about buttons the user should click.

Reply:
Yes, this should be italicized too